.docker/docker_run_git.sh

```diff
@@ -1,5 +1,25 @@
 #!/bin/sh
 
+if [ $PS_ENABLE_SSL = 1 ]; then
+  if [ -f ./.docker/ssl.key ]; then
```

Comment:
Add a message if the file doesn't exist for debug purposes ?

Reply:
Done